src/mxarray.jl

```diff
@@ -173,55 +173,55 @@ const _mx_is_empty   = mxfunc(:mxIsEmpty)
 const _mx_is_struct  = mxfunc(:mxIsStruct)
 const _mx_is_cell    = mxfunc(:mxIsCell) 
 
-macro mxget_attr(fun, ret)
-    :(ccall($(fun)::Ptr{Void}, $(ret), (Ptr{Void},), mx))
+macro mxget_attr(fun, ret, mx)
+    :(ccall($(esc(fun))::Ptr{Void}, $(esc(ret)), (Ptr{Void},), $(esc(mx))))
```

Comment:
The type assertion is not necessary although the use is also not precompilable. The right way is to use a global const `Ref` and initialize it in `__init__`.